doc/nrf/gs_modifying.rst

```diff
@@ -249,14 +365,14 @@ To select the build type when building the application from command line, specif
 .. parsed-literal::
    :class: highlight
 
-   -- -DCMAKE_BUILD_TYPE=\ *selected_build_type*\
+   -- -DCONF_FILE=prj_\ *selected_build_type*\.conf
 
-For example, you can replace the *selected_build_type* variable to build the ``ZRelease`` firmware for PCA20041 by running the following command in the project directory:
+For example, you can replace the *selected_build_type* variable to build the ``release`` firmware for ``nrf52840dk_nrf52840`` by running the following command in the project directory:
 
 .. parsed-literal::
    :class: highlight
 
-   west build -b nrf52840dk_nrf52840 -d build_nrf52840dk_nrf52840 -- -DCMAKE_BUILD_TYPE=ZRelease
+   west build -b nrf52840dk_nrf52840 -d build_nrf52840dk_nrf52840 -- -DCONF_FILE=prj_release.conf
```

Comment:
Delete the chapter

"Selecting a build type from command line"

We have already stated that the build type is set by setting CONF_FILE.

Reply:
Let's hold off with this change until after release. There are application README files that include these lines, and I prefer not to start a domino effect where we would have to review those README files as part of this PR.

Reply:
ok